api/Dockerfile

```diff
@@ -1,11 +1,13 @@
 FROM node:19.3-alpine
 
-WORKDIR /backend
+WORKDIR /api
 
-COPY . .
+COPY package*.json .
+
+RUN npm ci --omit=dev
 
-RUN npm ci --production
+COPY . .
 
 RUN npm run build
 
-CMD ["node", "dist/main"]
\ No newline at end of file
+CMD ["npm", "run", "start:prod"]
```

Comment:
You can use stage build to limit final docker image size

Reply:
done, that's reduce size and speed of building time :)